src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMultiSignMessage.cs

```diff
@@ -532,6 +971,20 @@ public void RemoveSignature(CoseSignature signature)
             _signatures.Remove(signature);
         }
 
+        /// <summary>
+        /// Removes the signature at the specified index from the message.
+        /// </summary>
+        /// <param name="index">The zero-based index of the signature to remove.</param>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///   <para>
+        ///     <paramref name="index"/> is less than 0.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     <paramref name="index"/> is equal to or greater than <see cref="Signatures"/> <see cref="ReadOnlyCollection{T}.Count"/>.
```

Comment:
Should there be a dot here?

```suggestion
        ///     <paramref name="index"/> is equal to or greater than <see cref="Signatures"/>.<see cref="ReadOnlyCollection{T}.Count"/>.
```

Reply:
I think it could also be an apostrophe and sound more English?
```suggestion
        ///     <paramref name="index"/> is equal to or greater than <see cref="Signatures"/>' <see cref="ReadOnlyCollection{T}.Count"/>.
```

Reply:
Seems reasonable to me.

Reply:
Or you could reword it to:
```suggestion
        ///     <paramref name="index"/> is equal to or greater than the number of elements in <see cref="Signatures"/>.
```